.github/workflows/cruft-prs.yml

```diff
@@ -0,0 +1,39 @@
+name: make cruft PRs for all projects using us
+on:
+  release:
```

Comment:
I'd also add workflow_dispatch

Reply:
Hmm, then we’d have to add all the release metadata used in the PR title body as `input`s

Reply:
OK, with the script it’s easy